spec/examples.txt

```diff
@@ -0,0 +1,25 @@
+example_id                           | status  | run_time               |
+------------------------------------ | ------- | ---------------------- |
+./spec/models/comment_spec.rb[1:1]   | passed  | 0.01382 seconds        |
+./spec/models/like_spec.rb[1:1]      | passed  | 0.00688 seconds        |
+./spec/models/post_spec.rb[1:1]      | passed  | 0.00158 seconds        |
+./spec/models/post_spec.rb[1:2]      | passed  | 0.00815 seconds        |
+./spec/models/post_spec.rb[1:3]      | passed  | 0.00129 seconds        |
+./spec/models/post_spec.rb[1:4]      | passed  | 0.00351 seconds        |
+./spec/models/user_spec.rb[1:1]      | passed  | 0.11147 seconds        |
+./spec/models/user_spec.rb[1:2]      | passed  | 0.20248 seconds        |
+./spec/requests/posts_spec.rb[1:1:1] | passed  | 0.44843 seconds        |
+./spec/requests/posts_spec.rb[1:1:2] | passed  | 0.30791 seconds        |
+./spec/requests/users_spec.rb[1:1:1] | passed  | 0.07724 seconds        |
+./spec/requests/users_spec.rb[1:1:2] | passed  | 0.05755 seconds        |
+./spec/views/user_spec.rb[1:1:1]     | failed  | 2 minutes 14.4 seconds |
+./spec/views/user_spec.rb[1:1:2]     | failed  | 2 minutes 13.7 seconds |
+./spec/views/user_spec.rb[1:1:3]     | unknown |                        |
+./spec/views/user_spec.rb[1:1:4]     | unknown |                        |
+./spec/views/user_spec.rb[1:2:1]     | unknown |                        |
+./spec/views/user_spec.rb[1:2:2]     | unknown |                        |
+./spec/views/user_spec.rb[1:2:3]     | unknown |                        |
+./spec/views/user_spec.rb[1:2:4]     | unknown |                        |
+./spec/views/user_spec.rb[1:2:5]     | unknown |                        |
+./spec/views/user_spec.rb[1:2:6]     | unknown |                        |
+./spec/views/user_spec.rb[1:2:7]     | unknown |                        |
```

Comment:
- [ ] Kindly make sure all tests for your **views** are **passing**. When trying on my side no one of them has passed and if you can check for the result above no test for views has passed.
Below is the result I get from your views tests(i.e.: `rspec spec/views `)
![Screenshot from 2022-11-16 16-11-13](https://user-images.githubusercontent.com/83042742/202206677-5de2a2d6-fa8b-477a-8182-aec6ba8d7c26.png)